stakingModel/contracts/UBIScheme.sol

```diff
@@ -292,12 +286,19 @@ contract UBIScheme is AbstractUBI {
         returns (bool)
     {
         uint256 claimers = 0;
-        for(uint256 i = 0; i < accounts.length && i < nextDayDistributionCount[currentDay]; ++i) {
-            if (nextDayDistributionAddresses[currentDay][accounts[i]].isEntitled &&
-                !nextDayDistributionAddresses[currentDay][accounts[i]].isCompleted) { // avoiding of double claiming
-                nextDayDistributionAddresses[currentDay][accounts[i]].isCompleted = true;
-                require(_claim(accounts[i]), "distributed claim was failed");
-                claimers = claimers.add(1);
+        for(uint256 i = 0; i < accounts.length && i < activeUsersCount; ++i) {
```

Comment:
what does activeUsersCount is? why do have this condition?

Reply:
we can do something else: we can check the gasleft and limit the iterations number so we won't be out of gas (also we can keep some "extra" gas). we need to do so as in the fish pool function.

Reply:
still activeUsersCount is irrelevant. maybe there's 1 activeuser and that active address is in account[50]
we dont need it

Reply:
updated.
89bbc9109fbba53e9fc6320604bab47dcd85de87